hawkrest/management/commands/hawkrequest.py

```diff
@@ -51,22 +73,25 @@ def handle(self, *args, **options):
         headers = {'Authorization': sender.request_header,
                    'Content-Type': request_content_type}
 
-        do_request = getattr(requests, method.lower())
-        res = do_request(url, data=qs, headers=headers)
+        try:
+            res = request(url, method.lower(), data=qs, headers=headers)
+        except ImportError:
+            raise CommandError('To use this command you first need to '
```

Comment:
I'd like to see this try/except/raise block moved back to the exact `import requests` statement within `request()` because otherwise it could obscure other imports.

For testing purposes you could create a `get_requests_module()` that does the import so you can mock it out.